packages/forklift-console-plugin/src/modules/Plans/planActions.tsx

```diff
@@ -258,6 +240,22 @@ const DeleteModal = ({
   }, [closeModal, setIsOpen]);
 
   const deletePlanMutation = useDeletePlanMutation(plan.namespace, exit);
+
+  const isPlanExecuting =
```

Comment:
1. `object` prop is an escape hatch to work with legacy code. According to the current design we should add this  flag to FlatPlan interface (pre-calculate) and re-use in other places.
2.  If we really need to reach directly then IMHO destructuring or custom small function is preferred to jsonpath

Reply:
1. I want to re-think that ... maybe not in this PR though 
2. no... I want to move to using jsonpath as a the default, it's a k8s standart that is common to the CLI, go code, OpenAPI, CRDs, I want us to use jsonpath too

but I agree, moving to use jsonpath may need it's own PR